azure-resources/ContainerService/managedClusters/kql/26ebaf1f-c70d-4ebd-8641-4b60a0ce0094.kql

```diff
@@ -8,4 +8,4 @@ PolicyResources
 | extend
     id = tostring(properties.resourceId)
 | summarize count() by id
-| project recommendationId="26ebaf1f-c70d-4ebd-8641-4b60a0ce0094", name, id, param1=strcat("numNonCompliantAlerts: ", count_)
+| project recommendationId="26ebaf1f-c70d-4ebd-8641-4b60a0ce0094", id, param1=strcat("numNonCompliantAlerts: ", count_)
```

Comment:
We'll probably need to parse the policy for the name column out of the resource ID versus deleting all together for WARA scripts.